prusti/tests/pass/parse/after_expiry.stdout

```diff
@@ -0,0 +1,30 @@
+// compile-flags: -Zprint-desugared-specs -Zprint-typeckd-specs -Zskip-verify -Zhide-uuids
+// normalize-stdout-test: "[a-z0-9]{32}" -> "$(NUM_UUID)"
+// normalize-stdout-test: "[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}" -> "$(UUID)"
+
+#![feature(register_tool)]
+#![register_tool(prusti)]
+#[prelude_import]
+use std::prelude::v1::*;
+#[macro_use]
+extern crate std;
+
+use prusti_contracts::*;
+
+fn prusti_pledge_item_test1_$(NUM_UUID)() { }
```

Comment:
Why this is empty?

Reply:
This is just a dummy function so that I can test the parsing, I don't really know what to actually generate.

Reply:
You have assertions in the pledge, so just follow the approach used for `requires` and `ensures`.

Reply:
Well, not only. What about the expression on the lhs of `=>`?

Reply:
You should be able to handle it in the same way as any other Rust expression inside an assertion.